corehq/apps/accounting/models.py

```diff
@@ -1885,7 +1885,7 @@ class InvoiceBase(models.Model):
     date_created = models.DateTimeField(auto_now_add=True)
     is_hidden = models.BooleanField(default=False)
     tax_rate = models.DecimalField(default=Decimal('0.0000'), max_digits=10, decimal_places=4)
-    balance = models.DecimalField(default=Decimal('0.0000'), max_digits=10, decimal_places=4)
+    balance = models.DecimalField(default=Decimal('0.0000'), max_digits=10, decimal_places=2)
```

Comment:
I like that we are now prepared for those $10M invoices.

Also, might want to change the default to only have 2dp?